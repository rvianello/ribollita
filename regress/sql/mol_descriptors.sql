SELECT mol_amw(mol_from_smiles('CCN)'));
SELECT mol_amw(mol_from_smiles('CC(=O)O'));
SELECT mol_amw(mol_from_smiles('c1ccccc1'));
SELECT mol_amw(mol_from_smiles(NULL));
SELECT mol_amw(mol_from_smiles('xyz'));
SELECT mol_amw(mol_from_smiles(''));
SELECT mol_amw(mol_from_smiles('C1CCCCC1'));


