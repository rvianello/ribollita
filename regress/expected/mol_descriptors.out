SELECT mol_amw(mol_from_smiles('CCN)'));
WARNING:  could not construct molecule
 mol_amw 
---------
        
(1 row)

SELECT mol_amw(mol_from_smiles('CC(=O)O'));
      mol_amw      
-------------------
 60.05200000000001
(1 row)

SELECT mol_amw(mol_from_smiles('c1ccccc1'));
      mol_amw      
-------------------
 78.11399999999999
(1 row)

SELECT mol_amw(mol_from_smiles(NULL));
 mol_amw 
---------
        
(1 row)

SELECT mol_amw(mol_from_smiles('xyz'));
WARNING:  could not construct molecule
 mol_amw 
---------
        
(1 row)

SELECT mol_amw(mol_from_smiles(''));
 mol_amw 
---------
       0
(1 row)

SELECT mol_amw(mol_from_smiles('C1CCCCC1'));
 mol_amw 
---------
  84.162
(1 row)

